### DISCUSSION TITLE
Raspberry Pi 3 and WS2801

### DISCUSSION THREAD

**Discussion by pappaq** [awawa-dev/HyperHDR#674] [Q&A]
Hey there,

I've searched the web for a configuration of a Raspberry 3 and a WS2801. I can't find anything, only that it is recommended and should be easy to set up.

I am in the process of migrating from an old config (Hyperion) for SDR content and FullHD to HDR/DV 4k (HyperHDR). At the moment, however, I am already failing to set up the WS2801 LED strip on the Raspberry Pi. I can't find anything about the GPIO configuration. The LED strip is connected to the Raspberry as follows. Unfortunately I can't get the LED strip to light up. Can someone tell me if the LED Strip is connected correctly for HyperHDR?

![image](https://github.com/awawa-dev/HyperHDR/assets/18289940/efd4eb04-b4ab-4c26-992c-2aa68c93361e)

Thanks in advance!

**Comment by awawa-dev** [comment]
Hi
Ws2801 is SPi type LEDs so it uses the same connection as popular APA102. The diagram is OK. Why is it not working? Missing 3.3V to 5V level shifter, SPI is not enabled in the OS, wrong SPI speed/path or other driver misconfiguration...or broken LEDs or cable connection. You need investigate it yourself.

**Reply by pappaq** [reply]
Thank you for your quick answer! Since you have confirmed the diagram I can now go on and diagnose the configuration!

**Reply by awawa-dev** [reply]
OK, start by checking the HyperHDR logs.
